app/middleware/searchValidator.js

```diff
@@ -0,0 +1,33 @@
+const log = require('../lib/logger');
+const renderer = require('../middleware/renderer');
+const searchValidator = require('../lib/searchValidator');
+
+function setSearchLabel(res, search) {
+  if (search) {
+    // eslint-disable-next-line no-param-reassign
+    res.locals.searchLabel = 'Enter your GP surgery name';
```

Comment:
Is this label supposed to change from `Enter a surgery name` to `Enter a surgery name` when the user enters nothing to search with?